clickhouse_sqlalchemy/__init__.py

```diff
@@ -1,15 +1,10 @@
 
-from .ext.declarative import get_declarative_base
-from .orm.session import make_session
-from .sql import Table, MaterializedView, select
-
-
-VERSION = (0, 3, 2)
-__version__ = '.'.join(str(x) for x in VERSION)
```

Comment:
Both versions are referenced at least [here](https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/f52087844b86a9524c2295185ac29274b293d129/docs/conf.py#L19); I suppose we can leave them here & make a dynamic link in `pyproject.yaml` instead

Reply:
Yeah, I missed that, I have no experience with sphinx or mkdocs, so I'll investigate the means of poetry and autodocs integration, thank you.

You can coathor if you find any solution, I'll be glad.